connectors/src/connectors/slack/index.ts

```diff
@@ -33,17 +33,15 @@ export async function createSlackConnector(
       if (teamInfo.ok !== true) {
         return new Err(
           new Error(
-            `Could not get slack team info. Error message: ${
-              teamInfo.error || "unknown"
+            `Could not get slack team info. Error message: ${teamInfo.error || "unknown"
```

Comment:
not sure this will be accepted by eslint?

Reply:
eslint does not seem to complain.

Reply:
https://github.com/dust-tt/dust/actions/runs/4832676037/jobs/8611746779?pr=334 ?